pkg/route/config_test.go

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) The Thanos Authors.
 // Licensed under the Apache License 2.0.
 
-package receive
+package route
```

Comment:
I think this should go under `receive`. This doesn't seem generic enough.

Reply:
Let's rename the package to `receive-route`?

Reply:
Done